2019-05-Berlin/agenda.md

```diff
@@ -10,43 +10,55 @@ It would be fantastic if each one of the breakouts is facilitated by a person wh
 
 ✨ We're excited for Node.js' 5th round of collaboration IRL! Even better, we're excited that folks from the JSF will be joining us for the first time! ✨
 
-> __Location TBA__,
-(Street Address)
-Berlin, Germany
+##Courtyard by Marriot, Berlin City Center
+Axel-Springer-Str. 55
+10117 Berlin, Germany
 
 
 ## Collaborator Summit Day One: 30th May, 2019
 
-(Time window)
+08:00 - 18:00
 
 ### Agenda
 
-(Types of rooms available):
 
-* (details of main room)
-* (details of more rooms)
+
+* Munchen 1/2, 4 - breakout rooms
+  - setup - pre-set rounds for 35 people
+  - AV - projector, screen and a flip chart (working on getting more AV set up)
+* Munchen 3 - breakout room
+  - theater style, 25 people
+* Berlin - main room
+  - 100 people, blocks of 6 table blocks
+   - projector, screen, sound system and 4 speakers, mixing table, 1 microphone 
+ 
+
 
 _(Expected participation number)_.
 
 
 | Time          | Place        | Topic                                         | Speakers/Facilitators                     | Issue                                      | Slides                                    |
 |---------------|--------------|-----------------------------------------------|----------------------------------------|--------------------------------------------|-------------------------------------------|
-| | Main         | Intro, Sponsor Talk, Code of Conduct          | |                                            | |
-
+| | Main  | Intro, Sponsor Talk, Code of Conduct          | |                                            | |
```

Comment:
Why is there a sponsor talk in a collab summit?

Reply:
Because we accept sponsors for the event to cover for part of the costs. This is usually 5 minutes or less.

Reply:
Ok, I don't remember we had one last year.

Reply:
We had one in Vancouver. Netflix paid for the lunch.